core-ui/src/components/Extensibility/components/Externallink.js

```diff
@@ -0,0 +1,44 @@
+import { useGetPlaceholder } from 'components/Extensibility/helpers';
+import { Link } from 'fundamental-react';
+import { isNil } from 'lodash';
+import { jsonataWrapper } from '../helpers/jsonataWrapper';
+
+export const Externallink = ({ value, schema, structure, ...props }) => {
+  const { emptyLeafPlaceholder } = useGetPlaceholder(structure);
+
+  const linkFormula = structure.linkFormula;
+  const textFormula = structure.textFormula;
+
+  function jsonata(formula) {
+    try {
+      const expression = jsonataWrapper(formula);
+
+      expression.assign('root', props.originalResource);
+      expression.assign('item', value);
+
+      return expression.evaluate();
+    } catch (e) {
+      console.warn('Widget::shouldBeVisible error:', e);
+      return { error: e };
+    }
+  }
+
+  let href;
+  if (typeof value === 'string') {
+    href = value.substring(0, 8) === 'https://' ? value : `https://${value}`;
```

Comment:
it works well, but there's a special fn in js for that [value.startsWith('https://')](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith)